libproxy.patch

```diff
@@ -87,6 +87,10 @@ index b4424e6..1f2dd2a 100644
 +#define DEBIAN_LIB_ARCH_DIR "x86_64-linux-gnu"
 +#endif
 +
++#if defined __aarch64__
```

Comment:
```suggestion
+#elif defined __aarch64__
```

Reply:
Also, I see there's three possible defines according to qprocessordetection : `__aarch64__`, `__ARM64__` and `_M_ARM64`

Reply:
Macros __ARM64__ is not exist. __aarch64__ is ARM64, macros _M_ARM64 is can use for compile on MSVC Compiler (see: ¹). It's good idea but we can add also other platforms (see: ²).

¹ - https://docs.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-170
² - https://wiki.debian.org/Multiarch/Tuples

Reply:
If it doesn't exist, why is it used by Qt then?
https://github.com/qt/qtbase/blob/dev/src/corelib/global/qprocessordetection.h#L98

Reply:
Maybe overcoding. GNU GCC compiler not have __ARM64__, maybe this added for MacOS, because uname command say about arm64, but any other OS installed with Parallels say about aarch64. You can view source code: https://github.com/gcc-mirror/gcc/tree/master/gcc/config and also: https://sourceforge.net/p/predef/wiki/Architectures/

Reply:
ok, please apply the suggested change, squash the commits and give a reasonable name to your commit & PR